scripts/ProcessTemplates.ps1

```diff
@@ -0,0 +1,149 @@
+
+# This script transforms the repo into a json with the following structure:
+
+# Root
+#  |
+#  |- Workbooks (reportFolder)
+#        |- Performance (categoryFolder)
+#             |- Apdex (templateFolder)
+#                 |- en
+#                 |   |- settings.json
+#                 |   |- readme.md
+#                 |   |- Apdex.workbook
+#                 |- ko
+#                     |- settings.json
+#                     |- readme.md
+#                     |- Apdex.workbook
+# 
+# If the transformation is successful the json gets published as artifact

```

Comment:
"if the transformation is successful, the json gets published as an artifact"

published where as what?

Reply:
does it just copy the file to a directory?  it doesn't go anywhere off your machine?

Reply:
the build has an artifacts folder that's where it gets published. Once the PR to master is completed a the release is triggered, the release definition uploads the artifact(JSON) to a storage account

Reply:
can you document this part somewhere:

> Once the PR to master is completed a the release is triggered, the release definition uploads the artifact(JSON) to a storage account

there's going to be a connection between this git repo and an VSTS pipeline or whatever, and how that is configured, etc.  here is just a script, but nothing in the *repo* uses the pipeline, some external pipeline does.

Reply:
can you document this part somewhere:

> Once the PR to master is completed a the release is triggered, the release definition uploads the artifact(JSON) to a storage account

there's going to be a connection between this git repo and an VSTS pipeline or whatever, and how that is configured, etc.  here is just a script, but nothing in the *repo* uses the pipeline, some external pipeline does.